app/controllers/users_controller.rb

```diff
@@ -147,7 +147,7 @@ def update
         field_id = f.id.to_s
         next unless params[:user_fields].has_key?(field_id)
 
-        val = params[:user_fields][field_id]
+        val = custom_field_value(f)
```

Comment:
This is just my preference, and I know you didn't make it this way to start, but can you change `f` to `field` here? I really dislike abbreviations for short variables, they just make it so much harder for me to read!

Reply:
I feel the same as you. Also `val` could be `value` ;)